plugins/uf2/uf2.quirk

```diff
@@ -0,0 +1,136 @@
+[BLOCK\UUID_E478-FA50]
+Plugin = uf2
+Name = ColorHug2
```

Comment:
huh?

Reply:
Ohh it's a test device I used for testing. We probably don't want this in the proper PR, and by then some of the proper hardware I've ordered will have turned up :)

Reply:
I've tested this with a `Raspberry Pi RP2` -- unfortunately this device seems to have a random UUID :(

Reply:
Maybe it's better to go off something else like SMBIOS / DT for devices like rpi.

Reply:
Ohh, it's not running *on* the rpi, it's when the rpi is in "peripheral" mode, i.e. it's the device you're attaching to your Linux device, rather than the other way around.

Reply:
is there anything else in the USB descriptor you could go off maybe then?

Reply:
That's a good question. Will poke.

Reply:
I'm struggling a bit going from dev path e.g. /dev/sda1 to the USB device. Any ideas? There's /sys/class/block/sda/device/model but that seems pretty raw.

Reply:
Presumably you can't go from udev parents? Did you look at the udev database for any other links?

Reply:
So this lead down a rabbithole, which then involved rewriting the plugin, possibly for the better. New idea here: https://github.com/fwupd/fwupd/pull/4096